### DISCUSSION TITLE
Set COMPOSE_FILE in .env not possible?

### DISCUSSION THREAD

**Discussion by tombreit** [containers/podman-compose#740] [Q&A]
Dear all,

Basically, I want to keep the container-related files (eg. `container-compose.yml`, `Containerfile`) bundled in one directory (here: `<repo_root>/container/`) as much as possible.

Currently, however, this does not seem to be possible in my setup where I set the `COMPOSE_FILE` in my `.env`. I've prepared a minimal repository: https://github.com/tombreit/podmancompose-hello-world

```bash
└─ $ ▶ cat .env 
COMPOSE_FILE=container/container-compose.yml
└─ $ ▶ podman-compose up
podman-compose version: 1.0.6
['podman', '--version', '']
using podman version: 4.3.1
no compose.yaml, docker-compose.yml or container-compose.yml file found, pass files with -f
```

How do I get podman-compose to have only the `.env` file in the repository root and other container-related files in `<repo_root>/container`?

I know that I can set the compose file on the command line (`-f`, `--file`), but it would be nice if I could set this via `.env`.

Thank you very much for any hint or tip,
Thomas

**Comment by mokibit** [comment]
It is possible now, fixed by https://github.com/containers/podman-compose/pull/1506.
